packages/wp-story-editor/src/testUtils/renderWithTheme.js

```diff
@@ -21,7 +21,7 @@ import { render } from '@testing-library/react';
 import { ThemeProvider } from 'styled-components';
 import { theme } from '@googleforcreators/design-system';
 
-// eslint-disable-next-line react/prop-types
+// eslint-disable-next-line react/prop-types -- @todo
```

Comment:
Same here, we should completely disable the `react/prop-types` rule for files within `testUtils` folders, then we can completely remove this comment.